includes/admin/class-wc-stripe-payment-requests-controller.php

```diff
@@ -0,0 +1,65 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Admin page for UPE Customize Express Checkouts.
+ *
+ * @since 5.4.1
+ */
+class WC_Stripe_Payment_Requests_Controller {
+	public function __construct() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'admin_scripts' ] );
+		add_action( 'wc_stripe_gateway_admin_options_wrapper', [ $this, 'admin_options' ] );
+	}
+
+	/**
+	 * Load admin scripts.
+	 */
+	public function admin_scripts() {
+
+		wp_register_style(
+			'wc_stripe-payment-requests_customizer',
+			plugins_url( 'build/style-payment_requests_customizer.css', WC_STRIPE_MAIN_FILE ),
+			[],
+			$script_asset['version']
```

Comment:
`$script_asset` isn't defined here yet (it's defined on line 34).

Reply:
Should be ready to go 👍